core/inc/knoting/networking_types.h

```diff
@@ -0,0 +1,162 @@
+#pragma once
+#include <knoting/types.h>
+#include <yojimbo.h>
+
+namespace knot {
+using namespace yojimbo;
+
+constexpr int SERVER_PORT = 13189;
+constexpr int CLIENT_PORT = 12646;
+constexpr int MAX_CLIENTS = 6;
+const char* const SERVER_ADDRESS = "100.117.241.88";
+// const char* const SERVER_ADDRESS = "127.0.0.1";
+//^^^ FOR LOCAL HOST TESTING ^^^
+const double TICK = 1.0 / 30.0;
+
+struct GameConfig : public ClientServerConfig {
+    GameConfig() {
+        ClientServerConfig();
+        this->numChannels = 2;
+        this->channel[0].type = yojimbo::CHANNEL_TYPE_RELIABLE_ORDERED;
+        this->channel[1].type = yojimbo::CHANNEL_TYPE_UNRELIABLE_UNORDERED;
+        this->timeout = 5;
+        this->networkSimulator = false;
+        this->maxPacketSize = 1500;
+    }
+};
+
+template <typename Stream>
+bool serialize_vec2i(Stream& stream, vec2i& val) {
+    auto valX = (int16_t)val.x;
+    auto valY = (int16_t)val.y;
+    serialize_int(stream, valX, -32767, 32767);
```

Comment:
what is -32767 & 32767

is this max and min data size?

Reply:
It is, I've now changed them to -1 to 1 as this packs them much better